xml/Accessibility/IAccIdentity.xml

```diff
@@ -12,15 +12,18 @@
   </AssemblyInfo>
   <Interfaces />
   <Attributes>
-    <Attribute>
+    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2">
       <AttributeName>System.Runtime.InteropServices.ComConversionLoss</AttributeName>
     </Attribute>
-    <Attribute>
+    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2">
       <AttributeName>System.Runtime.InteropServices.Guid("7852B78D-1CFD-41C1-A615-9C0C85960B5F")</AttributeName>
     </Attribute>
-    <Attribute>
+    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2">
       <AttributeName>System.Runtime.InteropServices.InterfaceType</AttributeName>
     </Attribute>
+    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5">
+      <AttributeName>System.Runtime.InteropServices.InterfaceType(1)</AttributeName>
```

Comment:
this new entry looks a bit odd @dend @joelmartinez 

From what I see on MSDN, .net 3.5 was System.Runtime.InteropServices.InterfaceType and later versions InterfaceTypeAttribute(ComInterfaceType.InterfaceIsIUnknown). Shouldn't that be showing here?

Reply:
@mairaw the `(1)` is likely the value of `ComInterfaceType.InterfaceIsIUnknown`, and the formatter always drops the `Attribute` suffix on attributes, since you can omit that when you apply an attribute in the source code. 

As to why it didn't expand the enum name, that might be a different issue, but "strictly speaking" this seems correct.

Reply:
@joelmartinez but the later versions are the ones that had the parameter and earlier versions were parameterless. So something looks not quite right here.

Reply:
@mairaw I just loaded up and inspected the `Accessibility.dll` from the `netframework-2.0` folder.
![image](https://user-images.githubusercontent.com/90380/43443656-85b06968-946f-11e8-919d-e7a504db7fe6.png)
This is in the VSMac assembly browser, and you can see that it lists `[InterfaceType (1)]` :)

Reply:
You're right. MSDN has that for 2.0 as well. Can you check the 3.5 version since you have that assembly browser handy? That's the one I was looking at this morning on MSDN.

Reply:
Thank you for looking into this @joelmartinez.

Reply:
Of course! 👏